Composer/packages/client/src/recoilModel/DispatcherWrapper.tsx

```diff
@@ -10,11 +10,11 @@ import { BotAssets } from '@bfc/shared';
 import { useRecoilValue } from 'recoil';
 import isEmpty from 'lodash/isEmpty';
 
+import { dialogsState } from './selectors';
```

Comment:
I know you're trying to have minimal change impact, but if it is a selector it should probably be called a selector.